### PR TITLE
add project files

### PR DESCRIPTION
**Pull request by olalipa**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.credit-card-front {
+    width: 324px;
+    display: flex;
+    justify-content: flex-end;
+    padding-left: 20px;
+}
+
+.number {
+
+}
```
Comment (Solnick) [inline style.css]: When you create a PR the first thing you should do is self-review. If you would do it this kind of code would not be here ;)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>Credit Card</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rouge+Script&display=swap" rel="stylesheet"></head>
+<body>
+<div class="credit-card credit-card-front">
```
Comment (Solnick) [inline index.html]: indentation

**Review by Solnick — CHANGES_REQUESTED** [review]
image
There are major differences. Please make sure that it is pixel perfect.

File: style.css [hunk 3]
```diff
@@ -1,81 +1,89 @@
...
+    margin-bottom: 25px;
+}
+
+.credit-card-front p {
+    margin-bottom: 3px;
+    margin-top: 0;
+}
+
+.credit-card-back .number {
+    font-family: "Roboto";
```
Comment (Solnick) [inline style.css]: please remember about the fallback font

**Review by Solnick — CHANGES_REQUESTED** [review]
image
The 123 text is using different font than in the designs

File: style.css (hunk 3 above)
Comment (olalipa) [reply style.css]: Added, thank you!

File: index.html (hunk 2 above)
Comment (olalipa) [reply index.html]: Done

File: style.css (hunk 1 above)
Comment (olalipa) [reply style.css]: You are right, I'm sorry about that!

**Review by Solnick — APPROVED** [review]

**Pull request merged**